workflows/fastq_prep/fastq_prep.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+// before running, make sure the tmp directory is set as /tmp
+// > cd <work directory>
+// > export TMPDIR=/tmp
```

Comment:
I know `export TMPDIR=/tmp` is something we often set manually prior to running workflows with singularity involved; is it possible to pre-set this in the global nextflow.config file, e.g., 
```
process {
    beforeScript = 'TMPDIR=$(mktemp -d); export TMPDIR'
}
```
or, potentially as a per process option if necessary? 
```
process TOFASTQ {
    beforeScript = 'TMPDIR=$(mktemp -d); export TMPDIR'
```
If it's not a quick fix, can leave it manually for time being

Reply:
thank you! this is a great solution, I updated the TOFASTQ process to set TMPDIR=/tmp in the beforescript block, and it runs successfully regardless of what TMPDIR is set to for your bash global variable.